test/integration_test.go

```diff
@@ -109,7 +110,8 @@ func processRequest(t *testing.T, request *pbsubstreams.Request, moduleGraph *ma
 
 	req := pipeline.NewRequestContext(ctx, request, isSubRequest)
 
-	baseStoreStore, err := dstore.NewStore("file:///tmp/test.store", "", "none", true)
+	file := fmt.Sprintf("file://%s", filepath.Join(os.TempDir(), "test.store"))
+	baseStoreStore, err := dstore.NewStore(file, "", "none", true)
```

Comment:
Golang has a test facility to deal with temp dir in test, you can use it here (seems the original author didn't know about that either :D, I just told the team). Also the `file://` is superflous.
```suggestion
	baseStoreStore, err := dstore.NewStore(filepath.Join(t.TempDir(), "test.store"), "", "none", true)
```

Reply:
Done